Assets/TextAssets/TXT/AMemory.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+/* Class that defines 
+ * 
+*/
+
+[System.Serializable]
+public class AMemory {

```

Comment:
Would recommend against using prepositions in variable / class / method names. e.g. "A." Would recommend just calling this class memory